cmake/modules/HCT.cmake

```diff
@@ -101,7 +107,7 @@ function(add_hlsl_hctgen mode)
   endif()
 
   add_custom_command(OUTPUT ${temp_output}
-                     COMMAND ${Python3_EXECUTABLE}
+                     COMMAND ${PYTHON_EXECUTABLE}
```

Comment:
This seems unrelated to the intent of the PR? Was it an in-progress change for something local or is it necessary for disabling in-tree sources?

Reply:
Doh! I mis-rebased the patch from an out of date branch.